IaC/ps56.cd/init.groovy.d/cloud.groovy

```diff
@@ -186,43 +173,7 @@ initMap['min-centos-6-x64'] = '''
 '''
 initMap['min-centos-7-x64'] = initMap['micro-amazon']
 initMap['fips-centos-7-x64'] = initMap['micro-amazon']
-initMap['min-centos-6-x32'] = '''
```

Comment:
We shouldn't remove it as there still might be builds

Reply:
This worker configuration was removed, as it wasn't available at all:
https://github.com/Percona-Lab/jenkins-pipelines/blob/e43a894fb855e34a9e26233eb0cefc0091a193e2/IaC/ps56.cd/init.groovy.d/cloud.groovy#L322

My changes only cleans the cloud.groovy up and don't remove any workers